engine/gui/src/gui.h

```diff
@@ -653,18 +653,18 @@ namespace dmGui
      * Adds a font with the specified name to the scene.
      * @note Any nodes connected to the same font_name will also be connected to the new font. This makes this function O(n), where n is #nodes.
      * @param scene Scene to add texture to
-     * @param font_name Name of the font that will be used in the gui scripts
+     * @param font_name_hash Hash of the font name that will be used in the gui scripts
      * @param font The font to add
      * @return Outcome of the operation
      */
-    Result AddFont(HScene scene, const char* font_name, void* font, dmhash_t path_hash);
+    Result AddFont(HScene scene, dmhash_t font_name_hash, void* font, dmhash_t path_hash);
```

Comment:
I think it e sense to have one function that uses `dmhash_t` instead of creation one more funciton